mycroft/skills/wolfram_alpha/__init__.py

```diff
@@ -184,7 +186,15 @@ def handle_fallback(self, message):
 
             self.speak(response)
         else:
-            self.speak("Sorry, I don't understand your request.")
+            if len(others) == 1:
```

Comment:
Assuming changes on suggestion below, we'd have a different verification here. I'd become something like:
`if alternative:` (only one value, not an array anymore)